### PR TITLE
Adding mirrors.bytes.ua

### PR DESCRIPTION
**Pull request by reseni23**
Would like to add a public Mirror on AlmaLinux in central part of Ukraine
Thanks!

### CODE REVIEW

File: mirrors.d/mirrors.bytes.ua.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: mirrors.bytes.ua
+address:
+  http: http://mirrors.bytes.ua/almalinux/
+geolocation:
+  continent: Europe
+  country: UA
+  city: Vinnytsia
```
Comment (jonathanspw) [inline mirrors.d/mirrors.bytes.ua.yml:8]:
```suggestion
  city: Vinnytsia Oblast
```

Comment (reseni23) [reply mirrors.d/mirrors.bytes.ua.yml]: thanks for fix

**Comment by reseni23** [comment]
@jonathanspw looks good, can we merge it ?
the only thing that check still failing in geo name

**Review by reseni23 — COMMENTED** [review]
looks good

File: mirrors.d/mirrors.bytes.ua.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: mirrors.bytes.ua
+address:
+  http: http://mirrors.bytes.ua/almalinux/
+geolocation:
+  continent: Europe
+  country: UA
```
Comment (jonathanspw) [inline mirrors.d/mirrors.bytes.ua.yml:7]:
```suggestion
  country: UA
  sate_province:
```

File: mirrors.d/mirrors.bytes.ua.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  sate_province:
+  city: Vinnytsia Oblast
```
Comment (jonathanspw) [inline mirrors.d/mirrors.bytes.ua.yml:9]:
```suggestion
  city: Vinnytsia
```

File: mirrors.d/mirrors.bytes.ua.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+name: mirrors.bytes.ua
+address:
+  http: http://mirrors.bytes.ua/almalinux/
+geolocation:
+  continent: Europe
+  country: UA
+  sate_province:
```
Comment (jonathanspw) [inline mirrors.d/mirrors.bytes.ua.yml:8]:
```suggestion
  state_province:
```

**Comment by jonathanspw** [comment]
Thanks for the mirror!

**Pull request merged**